components/Categories.php

```diff
@@ -76,25 +76,16 @@ public function onRun()
      */
     protected function loadCategories()
     {
+        $categories = BlogCategory::with('posts_count')->getNested();
         if (!$this->property('displayEmpty')) {
-            $categories = BlogCategory::whereExists(function($query) {
-                $prefix = Db::getTablePrefix();
-
-                $query
-                    ->select(Db::raw(1))
-                    ->from('rainlab_blog_posts_categories')
-                    ->join('rainlab_blog_posts', 'rainlab_blog_posts.id', '=', 'rainlab_blog_posts_categories.post_id')
-                    ->whereNotNull('rainlab_blog_posts.published')
-                    ->where('rainlab_blog_posts.published', '=', 1)
-                    ->whereNotNull('rainlab_blog_posts.published_at')
-                    ->where('rainlab_blog_posts.published_at', '<', Carbon::now())
-                    ->whereRaw($prefix.'rainlab_blog_categories.id = '.$prefix.'rainlab_blog_posts_categories.category_id')
-                ;
-            });
-            $categories = $categories->getNested();
-        }
-        else {
-            $categories = BlogCategory::getNested();
+            $iterator = function($categories) use (&$iterator) {
```

Comment:
@Rike-cz Please add a space between `function` and the open bracket on this line and the one below.
```suggestion
            $iterator = function ($categories) use (&$iterator) {
```